examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -380,6 +421,29 @@ void AppTask::StartTimer(uint32_t aTimeoutInMs)
     mFunctionTimerActive = true;
 }
 
+#ifdef CONFIG_CHIP_NFC_COMMISSIONING
+int AppTask::StartNFCTag()
+{
+    // Get QR Code and emulate its content using NFC tag
+    uint32_t setupPinCode;
+    std::string QRCode;
+
+    int result = GetQRCode(setupPinCode, QRCode, chip::RendezvousInformationFlags::kBLE);
+    if (result) {
```

Comment:
Can we use `SuccessOrExit` or `VerifyOrExit` instead, and maintain just one return statement per function/method?